src/test/java/seedu/address/model/profile/NameTest.java

```diff
@@ -40,4 +42,16 @@ public void isValidName() {
         assertTrue(Name.isValidName("123456789 123456789 1234")); // 24 chars
         assertFalse(Name.isValidName("123456789 123456789 12345")); // 25 chars invalid
     }
+
+    @Test
+    public void compareTo_sameCapitalisationAndSpelling_isEqual() {
+        assertEquals(new Name("Aaron").compareTo(new Name("Aaron")), 0);
+        assertEquals(new Name("65aaron").compareTo(new Name("65aaron")), 0);
+    }
+
+    @Test
+    public void compareTo_differentCapitalisationSameSpelling_isNotEqual() {
+        assertNotEquals(new Name("aaron").compareTo(new Name("Aaron")), 0);
+        assertNotEquals(new Name("A").compareTo(new Name("AA")), 0);
```

Comment:
Will it be better to use `assertEquals`, instead of just saying it won't be zero, to make it more explicit.